rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -528,33 +539,31 @@ class Pickle private (settings: Settings, mtab: Mtab, sroot1: String, sroot2: St
         false
       }
     }
-    def name: Name = {
+    private def isToplevel: Boolean = cacheSymIsToplevel.computeIfAbsent(ssym, _isToplevel)
+    def _name(ssym: String): Name = {
```

Comment:
Is this called elsewhere?  Or just a way to temporarily move it out of the namespace?